style.css

```diff
@@ -0,0 +1,306 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+h1 {
+  font-size: 40px;
+  color: #172b4d;
+  font-weight: 700;
+  max-width: 300px;
+  margin-bottom: 16px;
+  margin-top: 68px;
+  line-height: 52px;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  color: #172b4d;
+  margin-top: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.logo {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  color: #6070ff;
+  position: relative;
+  text-decoration: none;
+  margin-top: 30px;
+}
+
+h4 {
+  font-weight: 700;
+  line-height: 15px;
+  letter-spacing: 0;
+  color: #7f8cff;
+}
+
+ul {
+  display: inline-block;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  background-image: url(img/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+  height: 100vh;
+}
+
+.subcontent {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+ul li {
+  margin-right: 10px;
+}
+
+.Union {
+  position: absolute;
+  left: 322px;
+  top: 27px;
+  border: 3px;
+}
+
+h2 {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  color: #172b4d;
+}
+
+h3 {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+h3 span {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #7a869a;
+}
+
+h5 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  color: #172b4d;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 30px auto;
+}
+
+.canopy-card {
+  background: #f7f7f9;
+  padding: 20px 0;
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  grid-auto-rows: auto;
+  grid-gap: 1rem;
+}
```

Comment:
- [ ] Kindly make sure the background of this whole cards container covered the width of the page 100% in order to match the Figma design as much as possible. :wink: 

  - ### YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/219607505-b4dd6bc3-1134-4307-a543-be8405aff65b.png)

  - ### FIGMA DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/219607589-be0f74a1-b1e9-411c-8603-73aaa6e0b347.png)